extensions/spark/kyuubi-spark-lineage/src/test/scala/org/apache/kyuubi/plugin/lineage/helper/SparkSQLLineageParserHelperSuite.scala

```diff
@@ -1246,22 +1260,43 @@ class SparkSQLLineageParserHelperSuite extends KyuubiFunSuite
         List(
           ("default.view_tst.k", Set("default.t1.a")),
           ("default.view_tst.b", Set("default.t1.b")))))
+      spark.sql("DROP VIEW IF EXISTS t2")
+    }
+  }
+
+  test("test for skip parsing permanent view") {
+    withTable("t1") { _ =>
+      SparkContextHelper.setConf(LineageConf.SKIP_PARSING_PERMANENT_VIEW_ENABLED, true)
```

Comment:
withSQLConf() {
}

Reply:
The Conf here refers to the Spark level, not the SQL level. `withSQLConf` does not take effect here.

Reply:
is there any reason to not support for SQL level ? it is actually a SQL feature

Reply:
Maintain consistency of all SQLs lineage parsing within a Spark APP and avoid making changes at the SQL level.